featurebyte/routes/registry.py

```diff
@@ -294,6 +339,17 @@ class CatalogId:
 app_container_config.register_class(CatalogId, force_no_deps=True)
 
 
+class Payload:
+    """
+    Another special placeholder class.
+    """
+
+
+# Manually initialized via tasks.
+app_container_config.register_class(UUID, force_no_deps=True, name_override="task_id")
+app_container_config.register_class(Progress, force_no_deps=True)
+app_container_config.register_class(Payload, force_no_deps=True)
```

Comment:
Maybe we could just register `featurebyte.schema.worker.task.base.BaseTaskPayload` here since it is just a placeholder (could remove the class above & add a comment to indicate it is just a placeholder).

Reply:
Good idea! Will update

Reply:
Could fix it in coming PR since it is not super critical.

Reply:
sounds good, will update in a follow-up with a couple other changes :)